client/src/pages/Location.js

```diff
@@ -2,34 +2,58 @@ import React from "react";
 import styled from "styled-components/macro";
 import PageHeadline from "../components/Header/PageHeadline";
 import placeholder from "../assets/placeholder.jpeg";
-import Card from "../components/Card/Card";
 import ImageCard from "../components/Card/ImageCard";
-import Placeholder from "../components/Placeholder/Placeholder";
-import { Link } from "react-router-dom";
+import mappath from "../assets/map.svg";
 
-const TourDiv = styled.div`
+import { Link, useParams } from "react-router-dom";
+import { useQuery } from "react-query";
+import { getLocationByName } from "../api/locations";
+import DetailCard from "../components/Card/LocationDetails";
+import PictureContainer from "../components/Image/Picture";
+
+const LocationDiv = styled.div`
```

Comment:
```suggestion
const LocationContainer = styled.div`
```

Reply:
Every page container is named consistently. Such as "HomeDiv", "TourDiv"," LocationDiv". I don't think I should change this